src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -186,9 +186,6 @@
   <data name="net_http_argument_empty_string" xml:space="preserve">
     <value>The value cannot be null or empty.</value>
   </data>
-  <data name="net_http_client_request_already_sent" xml:space="preserve">
-    <value>The request message was already sent. Cannot send the same request message multiple times.</value>
-  </data>
```

Comment:
Why is this being deleted?  HttpClient's enforcement of an HttpRequestMessage being sent only once existed long before the request telemetry stuff.

Reply:
https://github.com/dotnet/runtime/pull/41022#discussion_r472925858

I can revert it, motivation in the comment.

Reply:
> I can revert it

Thanks.  That doesn't make the bar at this point.

Reply:
Done.